tests/integration/customizations/s3/test_plugin.py

```diff
@@ -497,6 +497,18 @@ def test_download_encrypted_kms_object(self):
         with open(local_filename, 'r') as f:
             self.assertEqual(f.read(), contents)
 
+    def test_download_empty_object(self):
+        bucket_name = self.create_bucket()
```

Comment:
We should be able to just use `_SHARED_BUCKET` instead of creating a new bucket.